oak_restricted_kernel/src/syscall/mod.rs

```diff
@@ -31,7 +31,7 @@ use crate::{mm, snp_guest::DerivedKey};
 use alloc::boxed::Box;
 use core::{arch::asm, ffi::c_void};
 use oak_channel::Channel;
-use oak_dice::evidence::RestrictedKernelDiceData as DiceData;
+use oak_dice::evidence::Evidence;
```

Comment:
The restricted kernel should still serve `RestrictedKernelDiceData`, but it should contain `Evidence` and `ApplicationPrivateKeys`